react/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "vtex.styleguide",
   "dependencies": {
+    "global": "^4.4.0",
```

Comment:
Those dependencies were added by mistake, I suppose :)

Reply:
Yes! Not sure how it happened. Sorry about that 🤦
Solved